ckanext/spectrum/tests/test_actions_user_create.py

```diff
@@ -35,14 +35,16 @@ def spectrum_org():
 class TestCreateUser():
 
     def test_unit_without_autogeneration(self, mock_token_urlsafe):
-        next_action = mock.Mock()
+        user = factories.User()
+        next_action = mock.Mock(return_value=user)
+        ctx = {'user': user}
```

Comment:
I'd just call it `context` you don't save much by the abbreviation, and it will just make reading the code a little more confusing.

Reply:
But also, why is context['user'] required?

Reply:
Agreed, name changed.

In previous version, since 'name' was enough to return from function tests worked. Now I had to add some new mocks to make them work.